relay/http.go

```diff
@@ -68,12 +69,24 @@ func NewHTTP(cfg HTTPConfig) (Relay, error) {
 			}
 			rb = newRetryBuffer(b.BufferSize, DefaultInitialInterval, DefaultMultiplier, max)
 		}
+		var transport http.RoundTripper // nil - will be use DefaultTransport
+
+		if b.SkipTLSVerification {
```

Comment:
There's no need for the `if b.SkipTLSVerification`. This can be simplified to

```
transport := &http.Transport{
	TLSClientConfig: &tls.Config{
		InsecureSkipVerify: b.SkipTLSVerification,
	},
}
```